packages/matchbox/src/components/ComboBox/ComboBoxTextField.tsx

```diff
@@ -20,7 +19,21 @@ const StyledWrapper = styled('div')`
   ${margin}
 `;
 
-function ComboBoxTextField(props) {
+export type ComboBoxTextFieldProps = {
+  delimiter?: string;
+  error?: string;
+  errorInLabel?: boolean;
+  helpText?: React.ReactNode;
+  id: string;
+  itemToString?: (any) => string;
```

Comment:
did you mean `(item: any)` ?

Reply:
I guess you could type it to `string | { [k: string]: any }` but not sure if worth it